rfd/0016-dynamic-configuration.md

```diff
@@ -231,29 +230,55 @@ This logic implies Choices 2.B and 3.B.
    immediately.
 
 2. If called without `--force` and the fetched resource does not have
-   `origin:defaults` then print
+   `origin: defaults` then print an error:
    ```
-   error: non-default cluster auth preference already exists, use -f or --force
-   flag to overwrite
+   non-default cluster auth preference already exists, use -f or --force flag
+   to overwrite
    ```
 
 3. If called with `--force` and the fetched resource does not have `origin:
    config-file` then store the resource specified in the provided YAML file
    in the backend.
 
 4. If called with `--force` and the fetched resource has `origin: config-file`
-   then print
+   then print an error:
    ```
-   error: This resource is managed by static configuration. We recommend
-   removing configuration from teleport.yaml, restarting the servers and trying
-   this command again.
+   This resource is managed by static configuration. We recommend removing
+   configuration from teleport.yaml, restarting the servers and trying this
+   command again.
 
    If you would still like to proceed, re-run the comand with both --force and
    --confirm flags.
    ```
 
-   This requires adding support for `--confirm` flag (that works only in
-   conjunction with `--force`).  Invocation with both `--force` and `--confirm`
-   always commits the YAML resource.
+   Invocation with both `--force` and `--confirm` will temporarily replace the
```

Comment:
The resource has `origin: config-file` implying there is (or recently was) explicit configuration touching the resource in `teleport.yaml`. Assuming this `teleport.yaml` isn't removed until the next auth server init, it will take precedence and overwrite anything that's stored in the backend.

Reply:
gotcha, suggested rewording to make this clearer